webpack/assets/javascripts/react_app/components/UserJwt/components/UserJwtField.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { Grid, Alert, LoadingState } from 'patternfly-react';
+import PropTypes from 'prop-types';
+
+import { translate as __ } from '../../../common/I18n';
+import { STATUS } from '../../../constants';
+import CopyToClipboard from '../../common/CopyToClipboard/CopyToClipboard';
+
+const UserJwtField = ({ apiStatus, token }) => {
+  if (apiStatus === STATUS.ERROR) {
+    return (
+      <Grid.Row>
+        <Grid.Col md={6}>
+          <Alert>
+            {__(
+              'There was an error while creating the JWT token, try refreshing the page.'
+            )}
+          </Alert>
+        </Grid.Col>
+      </Grid.Row>
+    );
+  }
+
+  return (
+    <LoadingState loading={apiStatus === STATUS.PENDING}>
+      {token && (
+        <Grid.Row style={{ marginTop: '14px' }}>
+          <Grid.Col md={4}>
+            <div style={{ marginBottom: '8px' }}>
+              <p>
+                {__(
+                  'Following JWT can be used for the API authorization, like this:'
+                )}
+                <br />
+                <code>
+                  curl -X GET &apos;{`${window.location.hostname}/api/v2/hosts`}
```

Comment:
Can you use `foremanUrl` here?

Reply:
I can't because `foremanUrl('/users/jwt')` returns only `/users/jwt`.
@laviro Maybe it's a bug

Reply:
Right, `foremanUrl` is for getting to correct "root path" to the server, it's needed when you are doing ajax calls etc.. 

If I understand correctly, here you are explaining how to obtain it from the terminal, where you really need the `hostname`, so this is correct 👍

Reply:
I think you should wrap the `/api/v2/hosts` in foremanUrl so if the foreman url is `localhost/special_foreman/' then
`window.location.hostname = localhost`
and `foremanUrl('api_path') = special_foreman/api_path`
or is the prefix not used in API?

Reply:
True, you will probably need the combination of both, e.g.:
`${window.location.pathname}${foremanUrl('/api/v2/hosts')}`
Bonus points would be to make an helper for it :)